self-host/customize-deployment/service-accounts.mdx

```diff
@@ -0,0 +1,300 @@
+
+<Info>
+  Service accounts require an **Enterprise License Key**.
+</Info>
+
+## Overview
+
+Service accounts allow you to:
+
+- Authenticate API requests without using personal user credentials
+- Grant specific permission scopes (read, edit, or admin)
+- Set expiration dates for tokens
+- Track token usage and last access time
+- Rotate credentials without affecting user accounts
+
+## Enabling service accounts
+
+Add the following environment variable to your configuration:
+
+```bash
+SERVICE_ACCOUNT_ENABLED=true
+```
+
+After enabling, restart your Lightdash instance for the change to take effect.
+
+Once enabled, organization admins can access service accounts at **Settings** → **Organization** → **Service Accounts**.
+
+## Permission scopes
+
+Service accounts use a hierarchical scope system where higher-level scopes inherit all permissions from lower levels.
+
+| Scope | Description |
+| :---- | :---------- |
+| `org:read` | Read-only access to organization resources |
+| `org:edit` | Modify resources (includes `org:read` permissions) |
+| `org:admin` | Full administrative access (includes all permissions) |
+| `scim:manage` | Special scope for SCIM-based user provisioning and deprovisioning |
+
+## Creating a service account
+
+### Via UI
+
+1. Navigate to **Settings** → **Organization** → **Service Accounts**
+2. Click **Create Service Account**
+3. Configure the token:
+   - **Description**: Descriptive name for the token (e.g., "Production CI/CD Pipeline")
+   - **Expiration**: Optional expiration date (recommended for security)
+   - **Scopes**: Select one or more permission scopes
+4. Click **Create**
+5. **Copy the token immediately** - it will only be shown once
+6. Store the token securely in a secrets manager
+
+### Via API
+
+Service accounts can also be created programmatically using an admin user's credentials:
+
+```bash
+curl -X POST \
+  -H "Authorization: Bearer $ADMIN_SESSION_TOKEN" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "description": "CI/CD Pipeline",
+    "expiresAt": "2025-12-31T23:59:59Z",
+    "scopes": ["org:admin"]
+  }' \
+  "$LIGHTDASH_URL/api/v1/organizations/$ORG_UUID/service-accounts"
+```
+
+## Using service accounts
+
+### Authentication
+
+Service account tokens use the `Authorization` header with the token value directly:
+
+```bash
+curl -H "Authorization: ldsvc_abc123xyz..." \
+  "$LIGHTDASH_URL/api/v1/..."
+```
+
+### Token format
+
+Service account tokens are prefixed with `ldsvc_`:
+
+```
+ldsvc_abc123xyz789...
+```
+
+This prefix distinguishes them from:
+- Personal access tokens (`ldpat_`)
+- SCIM tokens (`scim_`)
+- OAuth tokens (`ldapp_`, `ldref_`)
+
+## Common use cases
+
+### CI/CD integration
```

Comment:
All these cases also apply to user PAT . And perhaps they are very specific. They can be useful though.